### PR TITLE
Process bar is added without any styling.

### PR DESCRIPTION
**Pull request by viveriveniversum**
Great teamwork with Burak.

### CODE REVIEW

**Comment by brkzmn** [comment]
We have added a process bar to questionPage. We will add some CSS.

File: public/style.css [hunk 1]
```diff
@@ -15,3 +15,17 @@
...
+#process-bar-element li{
+  border: 1px solid red;
+}
+
+.active{
+  background-color: tomato;
+}
+.disabled{
+  background-color: lightslategrey;
+}
```
Comment (EnsarDogan) [inline public/style.css:31]: CoW. Missing the last line.

File: src/data.js [hunk 2]
```diff
@@ -22,6 +22,7 @@ export const quizData = {
         b: 'var, const, let, function',
         c: 'var, let, const',
       },
+      prize : '$100',
```
Comment (EnsarDogan) [inline src/data.js]: MoS. We should match the prize with the index. because in our project we are gonna change the order of the questions then.

**Review by EnsarDogan — COMMENTED** [review]
Good luck

**Review by EnsarDogan — APPROVED** [review]
Well done

**Review by brkzmn — APPROVED** [review]

**Review by EnsarDogan — APPROVED** [review]
Well done

**Review by viveriveniversum — COMMENTED** [review]
Thanks

**Pull request merged**